src/client/actions/Market.ts

```diff
@@ -66,56 +62,19 @@ export default class Market {
      * @param type card / pack / sticker
      * @returns a Promise resolved with the response or rejected in case of error
      */
-    public async getListings(
-        templateId: number,
-        page = 1,
-        sort: string = null,
-        type: string = null
-    ): Promise<MarketListing[]> {
+    public async getListings(templateId: number, page = 1, sort?: string, type = 'card'): Promise<MarketListing[]> {
```

Comment:
Is the type even required to work ? if that's not the case move it to optional param

Reply:
![image](https://user-images.githubusercontent.com/11234273/80850158-d04d8100-8c1a-11ea-94ca-753d294a1db6.png)